test/__tests__/bmap_test.ml

```diff
@@ -0,0 +1,102 @@
+open Test
+
+module C = Containers
+(* Int64 module tests *)
+
+(* Intentionally not tested as Int64 module is just a wrapper around stdlib, this should be part of BS testing *)
+
+
+(* Map module tests *)
+module IntMap = C.Map.Make(Int64)
+
+let to_int64 i = match C.Int64.of_int i  with
+  | Some x -> x
+  | None -> raise (Failure "Corrupted implementation of Int64.of_int")
+
+let map0 = IntMap.add (to_int64 1) "one" IntMap.empty
+
+let () = test "map.size" (fun t -> 
```

Comment:
Isn't the "map." redundant here? When there's an error, it will print the name of the file alongside the name of the test.